ldes-server-port-fetch/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fetchdomain/services/PrefixAdderImpl.java

```diff
@@ -0,0 +1,91 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.fetchdomain.services;
+
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.Statement;
+import org.apache.jena.shared.PrefixMapping;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+//TODO extract to util
+public class PrefixAdderImpl implements PrefixAdder {
```

Comment:
I would not duplicate this class either.. This is full of complex logic and will definitely result in a bug sooner than later.